gemviz/bluesky_runs_catalog.py

```diff
@@ -78,13 +78,12 @@ def catalog(self):
     def catalogName(self):
         return self.parent.catalogName()
 
-    def splitter_moved(self, key, pos, index):
-        deadline = getattr(self, f"{key}_deadline", 0)
+    def splitter_moved(self, key):
```

Comment:
I want to check this (later).  

My concern: This is a slot. Isn't `pos, index` emitted by the signal this slot receives?  Even though this slot code ignores them, we should leave them as-is or switch to a `*args` to collect them.

Reply:
I see. Let me know.

Reply:
Here's a screen view with `*args, **kwargs` added.  There are values in `args` that correspond to the `pos, index` arguments emitted by the signal.

Reply:
![image](https://github.com/BCDA-APS/gemviz/assets/2279984/9707c7d6-a87b-474f-a048-0896657f49cd)

Reply:
Here is a screen view with the code in the PR now:

![image](https://github.com/BCDA-APS/gemviz/assets/2279984/1c7d3d66-9136-431b-abd2-0d06e4d98c6a)

Reply:
How come this code did not raise an exception?

Reply:
Interesting! I can put them back.

Reply:
Done.